lib/web/desktop.go

```diff
@@ -496,3 +507,18 @@ func (h *Handler) desktopAccessScriptInstallADCSHandle(w http.ResponseWriter, r
 	_, err := io.WriteString(w, scripts.DesktopAccessScriptInstallADCS)
 	return nil, trace.Wrap(err)
 }
+
+// sendTDPNotification sends a tdp Notification over the supplied websocket with the
+// error message of err.
+func sendTDPNotification(ws *websocket.Conn, err error, fatal bool) error {
```

Comment:
What if we just passed in a `Severity` instead of the `fatal` boolean? It would be less custom code here, and it would be obvious at the call site what the last argument is, instead of wondering what the bool is used for.